src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionOverviewViewController.swift

```diff
@@ -20,19 +20,20 @@ import Foundation
 import UIKit
 
 
-class ExposureSubmissionOverviewViewController: DynamicTableViewController, SpinnerInjectable {
+class ExposureSubmissionOverviewViewController: DynamicTableViewController, SpinnerInjectable, ExposureSubmissionCoordinatorViewController {
 
 	// MARK: - Attributes.
 
-	@IBAction func unwindToExposureSubmissionIntro(_: UIStoryboardSegue) {}
 	var service: ExposureSubmissionService?
 	var spinner: UIActivityIndicatorView?
+	var coordinator: ExposureSubmissionCoordinator?
 
 	// MARK: - Initializers.
 
-	required init?(coder aDecoder: NSCoder, service: ExposureSubmissionService?) {
-		self.service = service
-		super.init(coder: aDecoder)
+	required init?(coder: NSCoder, coordinator: ExposureSubmissionCoordinator, exposureSubmissionService: ExposureSubmissionService?) {
```

Comment:
why can the exposure submission service be optional in this controller?
there are some other vcs where this is the case as well. please double check, thanks

Reply:
you are right, we can pass it as non-optional.